FloripaBus/SharedModule.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Ninject.Modules;
+using System.Net.Http;
+
+namespace FloripaBus
+{
+	public class SharedModule : NinjectModule
+	{
+		public override void Load() 
+		{			
+			this.Bind<HttpClient> ().ToSelf ();
+
+			this.Bind<INavigationService> ().To<NavigationService> ();
+
+			this.Bind<IRouteRepository>().To<RouteRepository>();
+
+			this.Bind<MainViewModel> ().ToSelf ();

```

Comment:
Why didn't you choose to create an interface to inject the dependency on the view models instead of binding to the object itself like you did for the `RouteRepository` and `NavigationService`?